dynamiq/nodes/tools/sql_executor.py

```diff
@@ -11,10 +11,10 @@
 
 
 class SQLInputSchema(BaseModel):
-    query: str = Field(..., description="Parameter to provide a query that needs to be executed.")
+    query: str = Field("", description="Parameter to provide a query that needs to be executed.")
```

Comment:
Please use None for the optional param, not "":
`query: str | None = Field(None, description="...")`